package.json

```diff
@@ -54,31 +54,73 @@
     "@types/commander": "^2.12.2",
     "@types/configstore": "^4.0.0",
     "async-mutex": "^0.2.4",
+    "axios": "^0.19.0",
```

Comment:
Please use normal `fetch`, there's no need to use `axios` at all. I know I used it before, but if we can avoid bringing in lots of dependencies, that's good.

Reply:
I shall take all these packages out, they creeped in from following an online tutorial for swagger. Haven't had a chance to remove them yet.